var/spack/repos/builtin/packages/elsi/package.py

```diff
@@ -65,11 +68,18 @@ class Elsi(CMakePackage, CudaPackage):
     )
     variant(
         "internal_elpa_version",
-        default="2024",
-        values=("2024", "2023_11", "2023", "2021", "2020"),
+        default="2020",
+        values=(
+            "2020",
+            "2021",
+            conditional("2023", when="@:2.11"),
+            conditional("2023_11", when="@:2.11"),
+            conditional("2024", when="@:2.11"),
```

Comment:
Just FYI, this is equivalent

```suggestion
            conditional("2023", "2023_11", "2024", when="@:2.11"),
```

Reply:
Thanks for pointing this out!